workitem.go

```diff
@@ -114,6 +114,42 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 	})
 }
 
+// Reorder reorders the workitem
```

Comment:
Please, try to be more precise about the documentation and please try to avoid explaining actions by using the same words (e.g. here: `Reorder reorders [...]`). I know this documentation is probably automatically generated but anyways.